src/test/resources/schemas/accessRights.json

```diff
@@ -1,16 +1,11 @@
 {
   "$id": "accessRights.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "conditionsOfAccess",
-  "description": "Description if resource is open access.",
+  "title": "Open Access Marker",
+  "description": "This is a marker if resource is open access. It does not state if a resource is not due to missing definitive info in the MARC data.",
```

Comment:
Better:

"This is a marker when the resource is open access. It does not state when a resource is not open access due to missing definitive info in the MARC data."